src/rpc/blockchain.cpp

```diff
@@ -1337,7 +1337,7 @@ UniValue z_gettreestate(const UniValue& params, bool fHelp)
     }
 
     // orchard
-    {
+    if (Params().GetConsensus().NetworkUpgradeActive(chainActive.Height(), Consensus::UPGRADE_NU5)) {
```

Comment:
Did you close this bracket somewhere?

Reply:
NM, all good.

Reply:
Can you do the same thing for Sapling above? If the `z_gettreestate` requests a block pre-sapling, the same bug is present.